### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by mueller-ma**
Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

**Comment by mueller-ma** [comment]
@maniac103 I noticed that AS saves user-specific stuff in workspace.xml, so we have to exclude it from git.

File: mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt [hunk 1]
```diff
@@ -46,8 +46,8 @@ class BackgroundTasksManager : BroadcastReceiver() {
                 NotificationUpdateObserver.createNotificationChannels(context)
             }
             ACTION_RETRY_UPLOAD -> {
-                val retryInfos = intent.getParcelableArrayListExtra<RetryInfo>(EXTRA_RETRY_INFOS)
-                for (info in retryInfos) {
+                val retryInfo = intent.getParcelableArrayListExtra<RetryInfo>(EXTRA_RETRY_INFO)
+                for (info in retryInfo) {
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt]: Old spelling seems more correct (it's a list, hence plural makes more sense)

File: mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt [hunk 2]
```diff
@@ -97,7 +97,7 @@ class BackgroundTasksManager : BroadcastReceiver() {
         private val TAG = BackgroundTasksManager::class.java.simpleName
 
         internal const val ACTION_RETRY_UPLOAD = "org.openhab.habdroid.background.action.RETRY_UPLOAD"
-        internal const val EXTRA_RETRY_INFOS = "retryInfos"
+        internal const val EXTRA_RETRY_INFO = "retryInfo"
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt]: As above

File: mobile/src/main/java/org/openhab/habdroid/background/NotificationUpdateObserver.kt [hunk 3]
```diff
@@ -75,30 +75,30 @@ internal class NotificationUpdateObserver(context: Context) : Observer<List<Work
         // - failed
         val hasEnqueuedWork = latestInfoByTag.any { (_, info) -> info.state == WorkInfo.State.ENQUEUED }
         val hasRunningWork = latestInfoByTag.any { (_, info) -> info.state == WorkInfo.State.FAILED }
-        val failedInfos = latestInfoByTag.filter { (_, info) -> info.state == WorkInfo.State.FAILED }
+        val failedInfo = latestInfoByTag.filter { (_, info) -> info.state == WorkInfo.State.FAILED }
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/background/NotificationUpdateObserver.kt]: As above

File: mobile/src/main/java/org/openhab/habdroid/background/NotificationUpdateObserver.kt [hunk 4]
```diff
@@ -75,30 +75,30 @@ internal class NotificationUpdateObserver(context: Context) : Observer<List<Work
...
         val nm = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
 
-        if (failedInfos.isNotEmpty()) {
+        if (failedInfo.isNotEmpty()) {
             // show error notification
             val errors = ArrayList<CharSequence>()
-            val retryInfos = ArrayList<BackgroundTasksManager.RetryInfo>()
-            for ((tag, info) in failedInfos) {
+            val retryInfo = ArrayList<BackgroundTasksManager.RetryInfo>()
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/background/NotificationUpdateObserver.kt]: As above

File: mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt [hunk 5]
```diff
@@ -148,9 +148,9 @@ class WidgetImageView constructor(context: Context, attrs: AttributeSet?) : AppC
         scope = null
     }
 
-    fun setRefreshRate(msec: Int) {
+    fun setRefreshRate(sec: Int) {
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt]: This seems off, as the argument is given in milliseconds ... maybe name it `ms`?

File: mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt (hunk 1 above)
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt]: Yes, plural is better here, but the plural of information is information.

Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt]: OK, so how about either `retryInfoList` or simply
```
for (info in intent.getParcelableArrayListExtra<RetryInfo>(EXTRA_RETRY_INFO_LIST))
```
(same line of thought applies below)?

I really don't want variable names to be misleading, even if the misleading name is linguistically more correct.

File: mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt (hunk 5 above)
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt]: `ms` is better, I didn't know what `msec` means.

**Comment by mueller-ma** [comment]
I also removed the changes in all `values-*/strings.xml` files.

File: mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.kt [hunk 6]
```diff
@@ -106,7 +106,7 @@ class WebViewFragment : Fragment(), ConnectionFactory.UpdateListener {
         return false
     }
 
-    private fun loadWebsite(urlToLoad: String = urltoLoad) {
+    private fun loadWebsite(urlToLoad: String = this.urlToLoad) {
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.kt]: `this.` needed?

File: mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt [hunk 7]
```diff
@@ -148,9 +148,9 @@ class WidgetImageView constructor(context: Context, attrs: AttributeSet?) : AppC
         scope = null
     }
 
-    fun setRefreshRate(msec: Int) {
+    fun setRefreshRate(ms: Int) {
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt]: Thinking about it, maybe we could/should name this `refreshDelayMs` or something?

File: mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.kt (hunk 6 above)
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.kt]: Yes, previously it was `urlToLoad` (fun) vs `urltoLoad` (class).

**Pull request merged**